crowdin.yml

```
@@ -0,0 +1,5 @@
+files:
+  - source: /messages/en.json
+    translation: /%original_path%/%two_letters_code%.%file_extension%
+  - source: /my-orders-plugin-example/messages/en.json
+    translation: /%original_path%/%two_letters_code%.%file_extension%
```

```suggestion
    translation: /%original_path%/%two_letters_code%.%file_extension%
```

será que não tá faltando uma parte pro my-orders-plugin-example? porque pelo que entendi, só vai estar puxando a pasta messages e o arquivo de locale

---

Pelo que entendi do manual do crowdin, ele pega [todas as pastas antecessoras](https://support.crowdin.com/vcs-integrations-configuring-online/?q=%25original_path%25).